bot/utils/safety_api.py

```diff
@@ -0,0 +1,41 @@
+import aiohttp
+from urllib import parse
+from loguru import logger
+
+from bot.constants import SAFETY_API_ENDPOINT, SAFETY_IDS
+
+
+def key_sort(article_id: int):
+    try:
+        category_index = SAFETY_IDS.index(article_id)
+    except ValueError:
+        category_index = 2000  # Number which shall never be reached in articles
+    return category_index
+
+
+async def get_api(query_string: str = None, category: int = None):
+    url_params = {}
+    if query_string:
+        url_params["query"] = f"{{{query_string}}}"
```

Comment:
are the..quadra `{}` intentional?

Reply:
@vladfrangu DMA [does the same thing](https://github.com/DiscordDevelopmentDiscord/discord-moderator-wumpus/blob/main/bot/utils/dma_api.py#L54), for what it's worth. In my own testing of the ZD api it doesn't look necessary, but there may be behavior I don't know about.

Reply:
> 
> 
> @vladfrangu DMA [does the same thing](https://github.com/DiscordDevelopmentDiscord/discord-moderator-wumpus/blob/main/bot/utils/dma_api.py#L54), for what it's worth. In my own testing of the ZD api it doesn't look necessary, but there may be behavior I don't know about.

Main difference is `""` is treated as everything when removed.